cmd/image-builder/config.go

```diff
@@ -270,6 +272,24 @@ func loadGithubActionsGitState() (GitStateConfig, error) {
 			BaseCommitSHA:   os.Getenv("GITHUB_SHA"),
 			BaseCommitRef:   os.Getenv("GITHUB_REF"),
 		}, nil
+
+	case "schedule":
+		// There is nostruct for schedule event in github package
+		var payload struct {
+			Repo github.Repository `json:"repository,omitempty"`
+		}
+		err = json.Unmarshal(data, &payload)
+		if err != nil {
+			return GitStateConfig{}, fmt.Errorf("failed to parse event payload: %s", err)
+		}
+		return GitStateConfig{
+			RepositoryName:  *payload.Repo.Name,
+			RepositoryOwner: *payload.Repo.Owner.Login,
+			JobType:         "schedule",
+			BaseCommitSHA:   os.Getenv("GITHUB_SHA"),
+			BaseCommitRef:   os.Getenv("GITHUB_REF"),
```

Comment:
Before using a `GITHUB_EVENT_NAME` and `GITHUB_EVENT_PATH` env vars we check if they are present. Should be do the same for `GITHUB_SHA` and `GITHUB_REF`?

Reply:
Yeah, that sounds reasonable i ncase someone will try to run it outside github environment (or Github will change the name for soem reason).